src/common/components/Header/HeaderScreen.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import { withRouter, useHistory } from 'react-router'
+import { Link } from 'react-router-dom'
+import styles from './HeaderScreen.module.css'
+import routes from '../../constants/routes.json'
+import Logo from '../../assets/icons/ico-logo.svg'
+import addBtn from '../../assets/icons/ico-addbtn.png'
+import searchIcon from '../../assets/icons/ico-search.svg'
+import QuestionTag from '../../assets/icons/ico-question.svg'
+import BellIcon from '../../assets/icons/ico-bell.svg'
+import MessageIcon from '../../assets/icons/ico-msg.svg'
+import SettingIcon from '../../assets/icons/ico-setting.svg'
+import UserIcon from '../../assets/icons/ico-user.svg'
+import cn from 'classnames'
+
+import Icon from '../Icon'
+
+interface IMenuType {
+  name: string
+  routeName: string
+  currentRoute: string
+  style: string | undefined
+}
+
+const MenubarItem = ({ name, routeName, currentRoute, style }: IMenuType) => {
+  let isActive = false
+
+  if (currentRoute.endsWith('app.html') || currentRoute === routeName) {
+    isActive = true
+  }
+
+  let activeColorClass = ''
+
+  if (isActive) {
+    activeColorClass = styles.headermainactive
+  }
+
+  return (
+    <div
+      // className={[
+      //   styles.headermenuitem,
+      //   activeColorClass,
+      //   'mr-8px md:mr-13px lg:mr-28px',
+      //   style,
+      // ].join(' ')}
+      className={cn(
+        styles.headermenuitem,
+        activeColorClass,
+        'mr-8px md:mr-13px lg:mr-28px font-medium h-22px',
+        style,
+      )}
+    >
+      <Link to={routeName}>
+        <span className={activeColorClass}>{name}</span>
+      </Link>
+    </div>
+  )
+}
+
+const SearhBar = () => {
+  const placeholder = 'Search creator or NFT'
+  return (
+    <div className='flex relative'>
+      <img width='16' className={styles.searchIconPosition} src={searchIcon} />
+      <input
+        className=' placeholder-gray-b0 h-41px bg-gray-110 rounded-full pl-46px md:w-300px lg:w-300px xl:w-352px'
+        placeholder={`${placeholder}`}
+      />
+    </div>
+  )
+}
+
+interface Location {
+  pathname: string
+}
+
+interface PropsType {
```

Comment:
Please use the Type instead of Interface

Reply:
fixed